internal/gonut/cmd/push.go

```diff
@@ -208,18 +210,39 @@ func runSampleAppPush(app sampleApp) error {
 		return fmt.Errorf("unsupported delete setting: %s", deleteSetting)
 	}
 
+	// If summary is set to full, enable pinging to display it in the summary.
+	if summarySetting == "full" {
+		pingSetting = true
+	}
+
 	appName := text.RandomStringWithPrefix(app.appNamePrefix, 32)
 
 	directory, err := app.assetFunc()
 	if err != nil {
 		return err
 	}
 
-	report, err := cf.PushApp(app.caption, appName, directory, cleanupSetting)
+	report, err := cf.PushApp(app.caption, appName, directory, cleanupSetting, pingSetting)
 	if err != nil {
 		return err
 	}
 
+	// If ping setting is enabled but the summary is not set to full,
```

Comment:
This breaks with the logic that follows afterwards. For example if summary is `json`, then `gonut` would print a text first and JSON output later. The whole output would not be JSON then. Same applies to the YAML output style. The oneline or short output would be affected, too. I would opt for removing this new part here and move the details in the respective outputs, for example to add the status code results in the exported report data.